src/plugins/plugin-http.ts

```diff
@@ -42,6 +42,16 @@ function getSpanName(options: ClientRequestArgs|url.URL) {
   return options.hostname || options.host || 'localhost';
 }
 
+/**
+ * Returns whether the Expect header is on the given options object.
+ * @param options Options for http.request.
+ */
+function hasExpectHeader(options: ClientRequestArgs|
+                         url.URL): options is ClientRequestArgs {
+  // tslint:disable-next-line:no-any
+  return !!((options as any).headers && (options as any).headers.Expect);
```

Comment:
I think it makes more sense to type-cast as `ClientRequestArgs` rather than as `any`.

Reply:
Done